packages/core/src/base-axis-chart.ts

```diff
@@ -136,6 +142,107 @@ export class BaseAxisChart extends BaseChart {
 		}));
 	}
 
+	createSlider(){
+
+		//TODO get the name of the action bar for the chart
+		var divToInsert = "#actions-" + this.options.type
+		console.log(divToInsert)
+
```

Comment:
Use of `var` is not recommended. Use `const` or `let`. Preferably `const` everywhere